packages/cspell/src/fileHelper.ts

```diff
@@ -31,7 +31,15 @@ export async function readConfig(configFile: string | undefined, root: string |
 
 export interface FileInfo {
     filename: string;
-    text: string;
+    text?: string;
+}
+export interface FileResult {
+    fileInfo: FileInfo;
+    processed: boolean;
+    issues: Issue[];
```

Comment:
Issues currently return the entire document. That will make the cache explode for large files with issues.

Reply:
I am fine with removing the "doc" field from Issues. Context should be enough.

Reply:
Ok, I'll remove that field. Is that ok in terms of reverse compatibility?

Reply:
I don't think it is being used by any reporters.